roles/tackle/templates/secret-keycloak-db.yml.j2

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+data:
+  POSTGRES_DATABASE: {{ rhsso_db_name_b64 }} 
+  POSTGRES_EXTERNAL_ADDRESS: {{ rhsso_db_host_b64 }}
+  POSTGRES_HOST:
```

Comment:
Should we get rid of POSTGRES_HOST here, seems like for external only POSTGRES_EXTERNAL_ADDRESS is used correct?

Reply:
No. It is intentional and blanks the value inserted by the RHSSO operator when it uses its own DB.

Reply:
:heavy_check_mark: